### PR TITLE
Master wowl popover mcm

### PR DESCRIPTION
**Pull request by mcm-odoo**

### CODE REVIEW

File: addons/web/static/src/core/popover/popover_container.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    setup() {
+        this.state = useState({ displayed: false });
+        this.targetObserver = new MutationObserver(this.onTargetMutate.bind(this));
+
+        useExternalListener(window, "click", this.onClickAway, { capture: true });
+    }
+    mounted() {
+        this.targetObserver.observe(this.target.parentElement, { childList: true });
+    }
+    __destroy() {
```
Comment (ged-odoo) [inline addons/web/static/src/core/popover/popover_container.js]: can't you do this in willunmount?

File: addons/web/static/src/core/popover/popover_container.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+export class PopoverContainer extends Component {
+    setup() {
+        // do not include popover params in state to keep original popover props
+        this.popovers = {};
+        const { bus } = useService("popover");
+        this.bus = bus;
+        this.bus.on("ADD", this, this.add);
+        this.bus.on("REMOVE", this, this.remove);
+    }
+    __destroy() {
```
Comment (ged-odoo) [inline addons/web/static/src/core/popover/popover_container.js:72]: do you really need to override `__destroy`? and not simply `destroy`?

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by mcm-odoo** [comment]
robodoo retry

**Pull request closed without merging**